packages/@webex/plugin-meetings/src/constants.ts

```diff
@@ -1331,3 +1331,15 @@ export const MEETING_PERMISSION_TOKEN_REFRESH_REASON = 'ttl-join';
 
 // constant for named media group type
 export const NAMED_MEDIA_GROUP_TYPE_AUDIO = 1;
+
+export const NOISE_REDUCTION_EFFECT_STATS = {
```

Comment:
Should we move these to `statsAnalayzer/constants.ts`? And if we do, maybe use more applicable names like `NOISE_REDUCTION_MODE` and `VIRTUAL_BACKGROUND_MODE`?